web/templates/weightclass.frag

```diff
@@ -1,41 +1,41 @@
 <select id="weightclass">
   <option value="all" selected>All Classes</option>
-  <optgroup label="Traditional">
-    <option value="t44">-44</option>
-    <option value="t48">-48</option>
-    <option value="t52">-52</option>
-    <option value="t56">-56</option>
-    <option value="t60">-60</option>
-    <option value="t67.5">-67.5</option>
-    <option value="t75">-75</option>
-    <option value="t82.5">-82.5</option>
-    <option value="t90">-90</option>
-    <option value="t90+">90+</option>
-    <option value="t100">-100</option>
-    <option value="t110">-110</option>
-    <option value="t125">-125</option>
-    <option value="t140">-140</option>
-    <option value="t140+">140+</option>
+  <optgroup id="traditional" label="Traditional">
+    <option value="t44">-44 kg</option>
```

Comment:
I don't think these should say kg/lbs. Instead of threading the state through the value, could we just have a JS global that remembers whether the selector is in kg or lbs?